robot_smach_states/test/manipulation/test_manipulation.py

```diff
@@ -11,6 +11,17 @@
 from robot_smach_states.util import designators as ds
 
 
+def assert_any_first_argument(unit_test_case, calls, argument):
+    for call in calls:
+        if call[0][0] == argument:
+            return
+    raise AssertionError("co call in {} has first argument {}".format(calls, argument))
+
+
+def assert_first_argument(unit_test_case, call, argument):
+    unit_test_case.assertEqual(call[0][0], argument)
+
```

Comment:
You may wonder why these custom functions are necessary. That is because the existing tests were too strict. A call with arguments ('open') would not pass because we require additional arguments in this function. In practice these additional arguments are provided by default arguments.

Reply:
I think these functions are related to our mockbot, so might be better to move these to there.

Reply:
They are related to mock in general. So yes I can definitly move these to the mockbot.

Reply:
I may have gone overboard... 
see [14bc248](https://github.com/tue-robotics/tue_robocup/pull/1061/commits/14bc2486e91cc52f81d47c67b03fd333e4682d10)